app/bundles/CoreBundle/Token/TokenAttribute.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * @copyright   2016 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\CoreBundle\Token;
+
+/**
+ * CLass Match.
```

Comment:
Remove this comment as it's wrong but even if corrected it has no value as the same is written 2 lines bellow.